new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -259,60 +171,60 @@ public void testCreateMiiDomain() {
+        String.format("createSecret failed for %s", adminSecretName));
 
     // create encryption secret
     logger.info("Create encryption secret");
     String encryptionSecretName = "encryptionsecret";
-    assertDoesNotThrow(() -> createDomainSecret(encryptionSecretName, "weblogicenc",
-            "weblogicenc", domainNamespace),
-             String.format("createSecret failed for %s", encryptionSecretName));
-
-    // create the domain CR
-    createDomainResource(domainUid, domainNamespace, adminSecretName, REPO_SECRET_NAME,
-              encryptionSecretName, replicaCount);
-
-    // wait for the domain to exist
-    logger.info("Check for domain custom resource in namespace {0}", domainNamespace);
-    withStandardRetryPolicy
-        .conditionEvaluationListener(
-            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
-                    + "(elapsed time {2}ms, remaining time {3}ms)",
-                domainUid,
-                domainNamespace,
-                condition.getElapsedTimeInMS(),
-                condition.getRemainingTimeInMS()))
-        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
-
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(
+                                      encryptionSecretName,
+                                      domainNamespace,
+                            "weblogicenc",
+                            "weblogicenc"),
+                    String.format("createSecret failed for %s", encryptionSecretName));
+
+    // create the domain object
+    Domain domain = createDomainResource(domainUid,
+                                      domainNamespace,
+                                      adminSecretName,
+                                      REPO_SECRET_NAME,
+                                      encryptionSecretName,
+                                      replicaCount,
+                              MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG);
+
+    // create model in image domain
+    logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG);
+    createDomainAndVerify(domain, domainNamespace);
 
     // check admin server pod exists
     logger.info("Check for admin server pod {0} existence in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodCreated(adminServerPodName, domainUid, domainNamespace);
+    checkPodExists(adminServerPodName, domainUid, domainNamespace);
 
     // check managed server pods exist
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Check for managed server pod {0} existence in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkPodCreated(managedServerPrefix + i, domainUid, domainNamespace);
+      checkPodExists(managedServerPrefix + i, domainUid, domainNamespace);
```

Comment:
You don't really need this check, the podReady actually validates the existence of the pod and ready status

Reply:
done